### PR TITLE
updated *.ts.files

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: qlipper.desktop [hunk 1]
```diff
@@ -7,40 +7,25 @@ Type=Application
...
-Name[lt]=Qlipper
-Name[pt]=Qlipper
 Name[sr]=Кјулипер
+Name[it]=Qlipper
 Name[sr@ijekavian]=Кјулипер
 Name[sr@ijekavianlatin]=Kjuliper
 Name[sr@latin]=Kjuliper
-Name[sv]=Qlipper
 GenericName=Clipboard Tool
-GenericName[ca]=Eina de porta-retalls
```
Comment (palinek) [inline qlipper.desktop:22]: Why did you remove some of the translations?

File: ts/qlipper_ca.ts [hunk 2]
```diff
@@ -66,227 +70,267 @@
     <name>QlipperPreferencesDialog</name>
     <message>
         <location filename="../src/qlipperpreferencesdialog.ui" line="14"/>
+        <location filename="../build/ui_qlipperpreferencesdialog.h" line="274"/>
```
Comment (palinek) [inline ts/qlipper_ca.ts:73]: All these `build/*` additions are wrong. Please, remove them.

**Review by palinek — CHANGES_REQUESTED** [review]

File: ts/qlipper_ca.ts (hunk 2 above)
Comment (stefonarch) [reply ts/qlipper_ca.ts:73]: Will try.
